cmd/SplitGrantsGovXMLDB/handler.go

```diff
@@ -129,6 +124,9 @@ func handleS3EventWithConfig(cfg aws.Config, ctx context.Context, s3Event events
 func readRecords(ctx context.Context, r io.Reader, ch chan<- grantRecord) error {
```

Comment:
Changes in this function enable developers to set a `MAX_SPLIT_RECORDS` environment variable in order to limit the number of records that will be extracted from the source DB during invocation. Specifically, it helps developers test on a reasonable number of records instead of needing to process >78k records from grants.gov.